voyager_archive/templates/voyager_archive/item_display.html

```diff
@@ -0,0 +1,136 @@
+{% extends 'voyager_archive/base.html' %}
+
+{% block content %}
+<form name="search_archive" id="search_archive" method="POST" enctype="multipart/form-data">
+    {% csrf_token %}
+    <table>
+        {{ form.as_table }}
+    </table>
+    <br>
+    <button type="submit">Search</button>
+</form>
+<hr/>
+
+
+<div class="row">
+    <div class="column">
+        <h1>Basic and Acquisitions Information</h1>
```

Comment:
Remove `and Acquisitions` from this table title, just `Basic Information` is OK.